### PR TITLE
Add support for 1.14.x protocols

### PR DESCRIPTION
**Pull request by Persson-dev**
I tested joining with 1.14.x clients and it seems to work.

### CODE REVIEW

File: src/Protocol/Protocol.h [hunk 1]
```diff
@@ -346,7 +346,11 @@ class cProtocol
 		v1_13   = 393,
 		v1_13_1 = 401,
 		v1_13_2 = 404,  // TODO: this constant should be in WebServer
```
Comment (12xx12) [inline src/Protocol/Protocol.h:348]: Maybe move this comment somewhere usefull (The top maybe)

**Review by 12xx12 — COMMENTED** [review]
I'll have a look later and compile this. Since I am experiencing problems, you aren't

**Review by 12xx12 — COMMENTED** [review]
Thanks!

Aren't there any packets changed (usually there is one or so)

**Comment by Persson-dev** [comment]
It might be the case but the wiki (https://minecraft.fandom.com/fr/wiki/1.14.x) says that these are minor releases and the wiki protocol (https://wiki.vg) is not very clear.

**Review by bearbin — APPROVED** [review]
If we discover any packet differences, then we can fix it with this boilerplate. Looks fine.

**Comment by 12xx12** [comment]
I just wanted to say that it also works fine.

File: src/Protocol/Protocol.h [hunk 2]
```diff
@@ -330,6 +330,7 @@ class cProtocol
 		Pose
 	};
 
+	// TODO: these constants should be in WebServer
```
Comment (tigerw) [inline src/Protocol/Protocol.h:333]: hah this was an HTTP 404 joke, it doesn't make sense now xD

**Pull request merged**